src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift

```diff
@@ -219,7 +219,7 @@ extension HomeInteractor {
 
 	func setupRiskConfigurator() -> CollectionViewCellConfiguratorAny? {
 		let detectionIsAutomatic = detectionMode == .automatic
-		let dateLastExposureDetection = riskDetails?.exposureDetectionDate
+		let dateLastExposureDetection = store.riskCalculationResult?.calculationDate
```

Comment:
Is there a reason why riskDetails is not uptodate?

Reply:
`riskDetails` is actually nil for the `RiskState`s `.inactive` and `.detectionFailed`, because they don't have an associated `Risk`. That's why the date was also nil.